src/CabanaPD_Integrate.hpp

```diff
@@ -125,7 +133,11 @@ class Integrator
         Kokkos::RangePolicy<exec_space> policy( 0, v.size() );
         Kokkos::parallel_for( "CabanaPD::Integrator::Final", policy,
                               final_func );
+
+        _timer.stop();
     }
+
+    auto time() { return _timer.time(); };
```

Comment:
@streeve We do not have here "auto timeInit() { return 0.0; };"

Reply:
You're right, that would be better for consistency - added